exercises/practice/armstrong-numbers/ArmstrongNumbersTest.bat

```diff
@@ -0,0 +1,142 @@
+@echo off
+REM ---------------------------------------------------
+REM Armstrong Numbers Unit Testing
+REM 
+REM sUnit Testing Framework version: 0.2
+REM ---------------------------------------------------
+
+:Main
+    REM Initalize result variable
+    set "slug=ArmstrongNumbers"
+
+    CALL :Initialize
+
+    REM --------------------
+    REM Test Case Start \/\/
+    REM Resource: https://github.com/exercism/problem-specifications/blob/main/exercises/acronym/canonical-data.json
+    REM --------------------
+    set "expected=true"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: Zero is an Armstrong number."
+    CALL :Assert 0
+
+    set "expected=true"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: Single-digit numbers are Armstrong numbers."
+    CALL :Assert 5
+
+    set "expected=false"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: There are no two-digit Armstrong numbers."
+    CALL :Assert 10
+
+    set "expected=true"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: Three-digit number that is an Armstrong number."
+    CALL :Assert 153
+
+    set "expected=false"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: Three-digit number that is not an Armstrong number."
+    CALL :Assert 100
+
+    set "expected=true"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: Four-digit number that is an Armstrong number."
+    CALL :Assert 9474
+
+    set "expected=false"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: Four-digit number that is not an Armstrong number."
+    CALL :Assert 9475
+
+    set "expected=true"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: Seven-digit number that is an Armstrong number."
+    CALL :Assert 9926315
+
+    set "expected=false"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: Seven-digit number that is not an Armstrong number."
+    CALL :Assert 9926314
+    
+    REM -- Batch Scripting does not support 64-bit integers --
```

Comment:
Feel free to not implement test cases when they're not appropriate for the track. You can add `include = false` to the test case in the `tests.toml` file to document it.